stencil-library/src/components/details/details.tsx

```diff
@@ -9,7 +9,6 @@ import { ErrorState } from './utils';
 
 export class Details {
   @Prop() errorMessage: string;
-  @Prop() entity: { metadata: any };
 
   render() {
     return (
```

Comment:
@TiagoFuelber I left the export parts here for now - I see that all of the parts listed here are actually used in the sub-components from `details/utils` - maybe they need to be exported here so that those classes can be successfully targeted by the end user?

Reply:
@colganmeanor I think we can safely remove the ones not it use by `Details` and `ErrorState`, since it's only using `ErrorState`.

Most of these exported-parts will need to be exported in `PaymentDetails` since this one is using some f the `details/utils` components